mobile/lib/utils/openapi_extensions.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:openapi/api.dart';
+
+import 'tuple.dart';
+
+/// Extension methods to retrieve ETag together with the API call
+extension WithETag on AssetApi {
```

Comment:
Is this extension available for all of the endpoints in `AssetApi`, however, it is only applied, or make sense when using the `getAllAssets` endpoint?

Reply:
This extension is only implemented for the `getAllAssets` endpoint. We can use the `WithETag` namespace to add more functions for other endpoints if needed. I guess fetching the images (thumb,large,full) could be potential future candidates.